appcheck/firebase-appcheck/src/main/java/com/google/firebase/appcheck/ktx/FirebaseAppCheck.kt

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.appcheck.ktx
+
+import com.google.firebase.FirebaseApp
+import com.google.firebase.appcheck.AppCheckToken
+import com.google.firebase.appcheck.FirebaseAppCheck
+import com.google.firebase.components.Component
+import com.google.firebase.components.ComponentRegistrar
+import com.google.firebase.ktx.Firebase
+
+/** Returns the [FirebaseAppCheck] instance of the default [FirebaseApp]. */
+@Deprecated(
+  "com.google.firebase.appcheckktx.Firebase.appCheck has been deprecated. Use `com.google.firebase.appcheckFirebase.appCheck` instead.",
```

Comment:
`com.google.firebase.appcheckktx.Firebase.appCheck has been deprecated`

This should be

`com.google.firebase.ktx.Firebase.appCheck` has been deprecated

This is also wrong in the next sentence.

Reply:
Updated